### ISSUE TITLE
Opening files in JASP (browser)

### ISSUE THREAD

**Issue by ankmb1979** [jasp-stats/jasp-issues#624]
* JASP version: 0.9.0.1
 Windows
* OS name and version:
* Analysis: None, just opening data
* Bug description: Data files do not open in the browser/rollapp version of JASP
* Expected behaviour: When clicking on Open/Computer/Desktop to import the data file on the desktop, JASP does not correctly provide a window for opening a file. Here is a screen shot of what I get: https://www.screencast.com/t/Yx0VwnW4JCa 

I would expect to get something like this instead (from the JASP download): https://www.screencast.com/t/8X22XTj3

#### Steps to reproduce:

 In short, I am teaching my students how to use JASP and many cannot download it, so I offered the browser version, but unfortunately they are unable to open files. I myself cannot locate or open files in the browser version of JASP.

**Comment by ankmb1979** [comment]
I figured out how to open the file.
